kotlin-reference-server/src/main/kotlin/org/stellar/reference/di/EventConsumerContainer.kt

```diff
@@ -0,0 +1,39 @@
+package org.stellar.reference.di
+
+import org.apache.kafka.clients.consumer.ConsumerConfig
+import org.apache.kafka.clients.consumer.KafkaConsumer
+import org.apache.kafka.common.serialization.StringDeserializer
+import org.stellar.reference.event.EventConsumer
+import org.stellar.reference.event.processor.AnchorEventProcessor
+import org.stellar.reference.event.processor.InMemoryTransactionStore
+import org.stellar.reference.event.processor.NoOpEventProcessor
+import org.stellar.reference.event.processor.Sep6EventProcessor
+
+object EventConsumerContainer {
```

Comment:
Can we listen to webhooks instead of kafka events?
If webhook is not yet supported, can you add a ticket to get rid of listening to kafka events on the business server?

Reply:
Created ANCHOR-439, will make the change after this PR is merged.